benchmarks/before_block_capture_block_vs_yield.rb

```diff
@@ -8,15 +8,15 @@ def yield_n_times(n)
   before_n_times(n) { yield }
 end
 
-def capture_block_and_yield_n_times(n, &block)
+def capture_block_and_yield_n_times(n)
```

Comment:
You need to revert this @pirj, its one of the points of the benchmark.

Reply:
Eagle eye 👍 
Fixed.